README.md

```diff
@@ -1,3 +1,9 @@
 Hi this is my readme!
 Here's my addition to read me.
 Here's another addition to test what the hell we're doing.
+
+Even more text! Omg look at us go!
+
+Wow.
+
+Much edits.
```

Comment:
Maybe 'more edits' or 'lots of edits' here instead? Let's discuss.

Reply:
Changed it!